src/less/variables.less

```diff
@@ -223,28 +227,28 @@
 @state-info-border:           darken(spin(@state-info-bg, -10), 10%); // #afe4ee
 @state-warning-text:          @brand-warning;
 @state-warning-bg:            #fcf8e3;
-@state-warning-border:        darken(spin(@state-warning-bg, -10), 10%); // #f7e1b5
+@state-warning-border:        #f5d9a5;
 @state-danger-text:           @text-color-error;
 @state-danger-bg:             #f2dede;
 @state-danger-border:         darken(spin(@state-danger-bg, -10), 5%); // #ebccd1
 
 //== Account Switcher
 //
-@account-switcher-header-bg:          @gray-700;
-@account-switcher-bg:                 @gray-800;
+@account-switcher-header-bg:          @black;
+@account-switcher-bg:                 @switcher-bg;
 @account-switcher-text:               @gray-200;
-@account-switcher-border-color:       @gray-700;
-@account-switcher-active-bg:          @gray-900;
+@account-switcher-border-color:       @black;
+@account-switcher-active-bg:          @black;
```

Comment:
Similar confusion here with account-switcher-active-bg and switcher-current-bg. Can we restore account-switcher-active-bg as the active background color, set it's value to #000 here and remove switcher-current-bg? Or add #000 to our color palette?